pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -213,7 +214,23 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				authorizationapi.NewRule("create").Groups(buildGroup).Resources(authorizationapi.JenkinsPipelineBuildResource).RuleOrDie(),
 			},
 		},
-
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: StorageAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				authorizationapi.NewRule(readWrite...).Groups(kapiGroup).Resources("bindings", "endpoints", "persistentvolumes", "persistentvolumes/status",
```

Comment:
this role will have to be granted at the cluster scope to enable access to PVs. I would not anticipate giving the storage admin control over every endpoint and service in the cluster